### PR TITLE
EV 355: part II: update schemas and enforce required property

### PR DESCRIPTION
**Pull request by klemmpnx**
This PR:
- update OCPP 201 standardized schema via script:
- add variable WebsocketPongTimeout in schema (already used in ChargePoint / manually added to 
- corrects usage of required "SecurityCtrlrIdentity" in "SecurityCtrlr" (key must be property key not variable name)
- add new struct RequiredComponentVariable as subclass of ComponentVariable for required variables (via script)
- adjust usages in ChargePoint:
   -  get_latest_meter_value_filtered requires "RequiredComponentVariable" (no further adaptions required)
   -  `authorize_req` does not set cacheExpiryDateTime if AuthCacheLifeTime not set (in accordance with spec; this would have failed before if this non-required variable had not been given)

### CODE REVIEW

File: lib/ocpp/v201/charge_point.cpp [hunk 1]
```diff
@@ -1508,9 +1508,12 @@ AuthorizeResponse ChargePoint::authorize_req(const IdToken id_token, const std::
...
     response = call_result.msg;
-    response.idTokenInfo.cacheExpiryDateTime = DateTime(
-        date::utc_clock::now() +
-        std::chrono::seconds(this->device_model->get_value<int>(ControllerComponentVariables::AuthCacheLifeTime)));
+    auto auth_cache_life_time =
+        this->device_model->get_optional_value<int>(ControllerComponentVariables::AuthCacheLifeTime);
+    if (auth_cache_life_time.has_value()) {
+        response.idTokenInfo.cacheExpiryDateTime =
+            DateTime(date::utc_clock::now() + std::chrono::seconds(auth_cache_life_time.value()));
+    }
```
Comment (Pietfried) [inline lib/ocpp/v201/charge_point.cpp:1516]:
```suggestion
```

This should be fixed with: https://github.com/EVerest/libocpp/commit/463975f05a91af41e4b09934bda85f6507db368b

**Review by Pietfried — CHANGES_REQUESTED** [review]

**Comment by klemmpnx** [comment]
@Pietfried : applied your changes, rebased and tested again

**Review by Pietfried — APPROVED** [review]

File: config/v201/component_schemas/standardized/InternalCtrlr.json [hunk 2]
```diff
@@ -197,6 +197,7 @@
     "SupportedCiphers12": {
       "variable_name": "SupportedCiphers12",
       "characteristics": {
+        "valuesList": "ECDHE-ECDSA-AES128-GCM-SHA256,ECDHE-ECDSA-AES256,GCM-SHA384,AES128-GCM-SHA256,AES256-GCM-SHA384,TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256",
```
Comment (Pietfried) [inline config/v201/component_schemas/standardized/InternalCtrlr.json:200]:
```suggestion
        "valuesList": "ECDHE-ECDSA-AES128-GCM-SHA256:ECDHE-ECDSA-AES256-GCM-SHA384:AES128-GCM-SHA256:AES256-GCM-SHA384",
```

File: config/v201/component_schemas/standardized/InternalCtrlr.json [hunk 3]
```diff
@@ -213,6 +214,7 @@
     "SupportedCiphers13": {
       "variable_name": "SupportedCiphers13",
       "characteristics": {
+        "valuesList": "TLS_AES_256_GCM_SHA384,TLS_AES_128_GCM_SHA256",
```
Comment (Pietfried) [inline config/v201/component_schemas/standardized/InternalCtrlr.json:217]:
```suggestion
        "valuesList": "TLS_AES_256_GCM_SHA384:TLS_AES_128_GCM_SHA256",
```

**Review by Pietfried — CHANGES_REQUESTED** [review]
default ciphersuites have been changed in the meantime:
https://github.com/EVerest/libocpp/pull/274

**Review by Pietfried — APPROVED** [review]

**Pull request merged**